doc/setup/install.rst

```diff
@@ -160,6 +216,27 @@ RPM
 
     For other RHEL-based distribution, see the EPEL_ documentation.
 
+Community
+'''''''''
+
+Execute the following commands to install Odoo 9.0 Community on your server:
+
+.. code-block:: console
+
+    $ sudo yum install -y postgresql-server
+    $ sudo postgresql-setup initdb
+    $ sudo systemctl enable postgresql
+    $ sudo systemctl start postgresql
+    $ sudo yum-config-manager --add-repo=https://nightly.odoo.com/9.0/nightly/rpm/odoo.repo
+    $ sudo yum install -y odoo
+    $ sudo systemctl enable odoo
+    $ sudo systemctl start odoo
+
+Enterprise
+''''''''''
+
+To install Odoo 9.0 Enterprise, execute these commands:
```

Comment:
Still need to test it on Fedora, don't merge yet because this section might be wrong.